src/main/java/frc/robot/commands/ShooterTeleop.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.ShooterSystem;
+
+public class ShooterTeleop extends Command {
+    private final ShooterSystem shooterSystem;
+    private final XboxController xboxController;
+
+
+    public ShooterTeleop(ShooterSystem shooterSystem, XboxController xboxController){
+        this.shooterSystem = shooterSystem;
+        this.xboxController = xboxController;
+
+        addRequirements();
```

Comment:
```suggestion
        addRequirements(shooterSystem);
```
you need to mention the subysystem inside the parenthesis

Reply:
done